bundle.Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=cryostat-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
 LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.6.1+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.4.0+git
```

Comment:
I did `make generate manifests bundle` on current `main` and got the same `additionalPrinterColumns` patch chunk as below, but I didn't get any of these `builder=operator-sdk` changes. Which invocation causes these and are these labels/annotations meaningful?

Reply:
It's just from `make bundle`. You must be using a newer `operator-sdk` binary than I am. I'm using 1.5.0, but for some reason it reports as `1.4.0+git`. #325 changed these labels to their current values, this is just changing them back.

Reply:
I'm not sure if it serves a purpose beyond documenting the version the operator was built with.